binancial/strategies/weighted_random.py

```
@@ -0,0 +1,32 @@
+        based on weighted probabilities.
+
+        ticker | pandas dataframe | historical price data
+        accountant | Accountant object | account information
+        price_usdt | float | current price of the asset
+        timestamp | datetime | current timestamp
+        '''
+
+        import random
+
+        actions = ['buy', 'sell', 'hold']
+        action = random.choices(actions, weights=self.weights, k=1)[0]
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)